erpnext/setup/doctype/incoterm/incoterm.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe.model.document import Document
+
+
+class Incoterm(Document):
+	pass
+
+
+def create_incoterms():
+	"""Create Incoterm records from incoterms.csv."""
+	import os
+	from csv import DictReader
+
+	with open(os.path.join(os.path.dirname(__file__), "incoterms.csv"), "r") as f:
```

Comment:
Any specific reason for reading them from a file rather than simply hardcoding a dictionary/object in this file itself?

Reply:
Separating data from code, non-technical people can modify/extend it without having to know about python